static/src/javascripts/projects/common/modules/video/events.js

```diff
@@ -152,7 +135,8 @@ define([
             return 'media:' + eventName;
         }).forEach(function(playerEvent) {
             player.on(playerEvent, function(_, mediaEvent) {
-                ga(gaTracker + '.send', 'event', buildGoogleAnalyticsEvent(mediaEvent, events, canonicalUrl));
+                ga(gaTracker + '.send', 'event', gaHelper.buildGoogleAnalyticsEvent(mediaEvent, events, canonicalUrl,
+                    'guardian-videojs', getGoogleAnalyticsEventAction, mediaEvent.mediaId));
```

Comment:
@kelvin-chappell I agree but let's make this change in a separate PR as I don't want to change the PROD behaviour for hosted in this PR.